src/spyglass/common/common_interval.py

```diff
@@ -78,26 +79,47 @@ def fetch_interval(self):
         if not len(self) == 1:
             raise ValueError(f"Expected one row, got {len(self)}")
         return Interval(self.fetch1())
+        
+    def plot_intervals(self, start_time=0, return_fig=False):
+        """
+        Parameters
+        ----------    
+        start_time: int (seconds)
+            set the 0 time for your interval comparison plot (for example, the first timepoint of s1)
+
+        Returns
+        -------
+        None
+            this function displays a timeline plot of the intervals you want to compare
+        """
+        interval_lists_df = pd.DataFrame(self)
+        interval_list_names = interval_lists_df['interval_list_name'].values
+
+        n_compare = len(interval_list_names)
+
+        # plot broken bar horizontals
+        fig, ax = plt.subplots(figsize=(20, 2 / 3 * n_compare))
+
+        # get n colors
+        cmap = plt.get_cmap("turbo", n_compare)
+        custom_palette = [mpl.colors.rgb2hex(cmap(i)) for i in range(cmap.N)]
+
+        def convert_intervals_to_range(intervals, start_time):
+            return [
+                ((i[0] - start_time) / 60, (i[1] - i[0]) / 60)
+                for i in intervals
+            ]  # return time in seconds
```

Comment:
Time in spyglass is in `unix time` which is seconds since 1970. Is this intended to be minutes?